### PR TITLE
Add mostLikelyEmail util

### PR DESCRIPTION
**Pull request by eliasbraun**
In order to simplify basic client side email validation, it makes sense to outsource that into a util, which is the proposal you'll find in this PR.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -70,6 +70,27 @@ describe('validations', () => {
...
+      'a.b+c@example.com',
+      'a_b.c+d@example.com',
+      'asdasdasd@.',
+    ];
+
+    const falsy = [
+      'a.example.com',
+      '@',
+      'a@b',
+      'a@b.de',
```
Comment (rejas) [inline test/index.js:86]: that isnt allowed?

Comment (eliasbraun) [reply test/index.js:86]: No, because we want to ensure a min length of 7 chars. Behind the '@', there has to be at least two chars.

**Review by rejas — APPROVED** [review]

File: test/index.js (hunk 1 above)
Comment (peterhass) [reply test/index.js:86]: Didn't read the email spec but I still wonder if that's a valid requirement 🤷

**Review by peterhass — APPROVED** [review]

File: test/index.js (hunk 1 above)
Comment (eliasbraun) [reply test/index.js:86]: Yes, since a domain has at least 2 chars, and the part behind the the '@' has to be at least 2 chars. 
1+@+2+.+2 = 7

**Pull request merged**